examples/mobile/UIComponents/index.html

```diff
@@ -6,7 +6,7 @@
 	<title>
 		Tizen UI Components
 	</title>
-	<link href="lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="lib/tau/tv/theme/changeable/tau.css" rel="stylesheet" />
```

Comment:
It's examples/**mobile**/UIComponents.
But you change it "**tv**/them/changeable"
Is it correct?

Reply:
You are right! :) . You are very perceptive, I missed it. Thanks.